File: app/src/main/java/com/patrykkosieradzki/qrcodereader/utils/DateUtils.java
package com.patrykkosieradzki.qrcodereader.utils;


import java.text.SimpleDateFormat;
import java.util.Calendar;

public class DateUtils {

    public static String getCurrentDateAsString() {
        return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
                .format(Calendar.getInstance()
                        .getTime());
    }

}

File: app/src/main/java/com/patrykkosieradzki/qrcodereader/utils/DateUtils.kt
package com.patrykkosieradzki.qrcodereader.utils


import java.text.SimpleDateFormat
import java.util.Calendar

object DateUtils {

    val currentDateAsString: String
        get() = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
                .format(Calendar.getInstance()
                        .time)

}
